src/main/java/se/sundsvall/chat/api/model/Conversation.java

```diff
@@ -0,0 +1,171 @@
+package se.sundsvall.chat.api.model;
+
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.util.List;
+import java.util.Objects;
+
+@Schema(description = "Represents a conversation containing participants, messages, and metadata.")
+public class Conversation {
```

Comment:
Add `private List<String> externalReferences`

Reply:
externalReferens should be key valus (like metadata) if multiple reference with different types is needed. Map<String, List<String>>